services/brig/src/Brig/Calling/API.hs

```diff
@@ -96,20 +104,26 @@ handleNoTurnServers (Left NoTurnServers) = do
 
 getCallsConfig ::
   ( Member (Embed IO) r,
-    Member SFT r
+    Member SFT r,
+    Member GalleyAPIAccess r
   ) =>
   UserId ->
   ConnId ->
   (Handler r) Public.RTCConfiguration
-getCallsConfig _ _ = do
+getCallsConfig uid _ = do
   env <- view turnEnv
   discoveredServers <- turnServersV1 (env ^. turnServers)
+  shared <- do
+    ccStatus <- lift $ liftSem $ (wsStatus . afcConferenceCalling <$> getAllFeatureConfigsForUser (Just uid))
+    pure $ case ccStatus of
+      FeatureStatusEnabled -> True
+      FeatureStatusDisabled -> False
   eitherConfig <-
     (dropTransport <$$>)
       . lift
       . liftSem
       . Polysemy.runError
-      $ newConfig env discoveredServers Nothing Nothing Nothing HideAllSFTServers CallsConfigDeprecated
+      $ newConfig env discoveredServers Nothing Nothing Nothing HideAllSFTServers CallsConfigDeprecated shared
```

Comment:
Pretty sure that we don't create these credentials for the non-V2 endpoint. So, we could simply pass `False` and no test should fail.